tgui/docs/tutorial-and-examples.md

```diff
@@ -34,32 +36,36 @@ powerful interactions for embedded objects or remote access.
 
 ### Backend
 
-Let's start with a very basic hello world.
+Let's start with a very basic example. We'll use the computer object as a base to make it easier to test.
+
+A note about file location. This can be placed anywhere you'd put other DM code. The frontend must be placed in a specific place - more on that below.
 
 ```dm
-/obj/machinery/my_machine/ui_interact(mob/user, datum/tgui/ui)
+/obj/machinery/computer/TGUI_Test_Computer
```

Comment:
whys this capitalized

Reply:
because i'm a menace fikou
(i'll fix it next pass)